h2o-core/src/main/java/water/api/FramesHandler.java

```diff
@@ -69,7 +69,7 @@
         // Weed out frames with vecs that are no longer in DKV
         boolean skip = false;
         for( Vec vec : frame.vecs() ) {
-          if (DKV.get(vec._key) == null) {
+          if (vec == null || DKV.get(vec._key) == null) {
```

Comment:
If I understand well - this handles a case when `FramesHandler` is executed in the middle of frame update. 

Just thinking here - It is little bit suspicious that `vec` can be `null`, since the iterator in `for` holds reference to an array returned by `frame.vecs()` and somebody is modifying the array under the hood. 
If i look into `Frame` class, we mostly update the internal `vecs` array by creating a new array...